strapi/strapi
# fix: add support for private upload providers

### What does it do?

- Signs urls when they are coming from a private upload provider

### Why is it needed?

- users can't export and transfer files from private upload providers

### How to test it?

- create a private upload provider and attempt to export, import and transfer
- make sure that normal transfer work the same and are not affected by the changes

you can follow the example provided in this issue #17571 

### Related issue(s)/PR(s)

fix #17571
